README.md

```diff
@@ -218,6 +221,58 @@ if (entity != null) {
 }
 ```
 
+Google Cloud DNS (Alpha)
+----------------------
+- [API Documentation][dns-api]
+- [Official Documentation][cloud-dns-docs]
+
+*Follow the [activation instructions][cloud-dns-activation] to use the Google Cloud DNS API with your project.*
+
+#### Preview
+
+Here are two code snippets showing simple usage examples from within Compute/App Engine. Note that you must [supply credentials](#authentication) and a project ID if running this snippet elsewhere.
+
+The first snippet shows how to create a zone resource. Complete source code can be found on
+[CreateAndListZones.java](./gcloud-java-examples/src/main/java/com/google/gcloud/examples/dns/snippets/CreateAndListZones.java).
```

Comment:
This snippet doesn't quite line up to the content of CreateAndListZones (there's a list in the actual code)